core/arch/arm/kernel/generic_entry_a32.S

```diff
@@ -155,6 +155,45 @@ KEEP_INIT _start
 	.endm
 
 	/*
+	 * set_sctlr : Setup some core configuration in CP15 SCTLR
+	 *
+	 * Followng setup is required by current core implementation:
+	 * - Disable data and instruction cache.
+	 * - MMU is expected off and exceptions trapped in ARM mode.
+	 * - Enable or disable alignment checks.
+	 * - Optinally enable write-implies-execute-never.
+	 * - Optinally enable round robin strategy for cache replacement.
+	 * - Optinally enable program flow prediction.
+	 *
+	 * Clobbers r0.
+	 */
+	.macro set_sctlr
+		read_sctlr r0
+		bic	r0, r0, #(SCTLR_M | SCTLR_C)
+		bic	r0, r0, #SCTLR_I
+		bic	r0, r0, #SCTLR_TE
+#if defined(CFG_SCTLR_ALIGNMENT_CHECK)
+		orr	r0, r0, #SCTLR_A
+#else
+		bic	r0, r0, #SCTLR_A
+#endif
+#if defined(CFG_HWSUPP_MEM_PERM_WXN) && defined(CFG_CORE_RWDATA_NOEXEC)
+		orr	r0, r0, #(SCTLR_WXN | SCTLR_UWXN)
+#endif
+#if defined(CFG_ENABLE_SCTLR_RR)
+		orr	r0, r0, #SCTLR_RR
+#endif
+#if defined(CFG_ENABLE_SCTLR_Z)
+		/* Some ARMv7 architectures need btac flush with post synchro */
+		write_bpiall
+		dsb
+		isb
+		orr	r0, r0, #SCTLR_Z
+#endif
```

Comment:
Current proposal already handles that. In **core/arch/arm/cpu/cortex-armv8-0.mk**, you found
```makefile
$(call force,CFG_ENABLE_SCTLR_RR,n)
$(call force,CFG_ENABLE_SCTLR_Z,n)
```
So ARMv8 cores will not currently never try to enable SCTLR[RR] or SCTLR[Z].